<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>30</y>
      <width>421</width>
      <height>622</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_26">
     <item>
      <widget class="QLabel" name="label_51">
       <property name="styleSheet">
        <string notr="true">font-size: 16px;
font-weight: bold;
color: #000;</string>
       </property>
       <property name="text">
        <string>Companies list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="companiesTable_6">
       <property name="autoScrollMargin">
        <number>16</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::DashLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_54">
       <property name="styleSheet">
        <string notr="true">color: #000;
font-size: 16px;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Eligible employees</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="eligibleEmployeesList"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>371</width>
      <height>622</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_28">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_29">
       <item>
        <widget class="QLabel" name="label_57">
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-weight: bold;
color: #000;</string>
         </property>
         <property name="text">
          <string>Employees list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_55">
         <property name="styleSheet">
          <string notr="true">color: #000;
font-size: 16px;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Without job</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_16"/>
       </item>
       <item>
        <widget class="QLabel" name="label_60">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Delete a company</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Delete an employee</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_30">
       <property name="topMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QPushButton" name="addCompany_6">
         <property name="text">
          <string>Add a company</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addEmployee_6">
         <property name="text">
          <string>Add an employee</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
